src/shared/components/palette/complex/read-complex-field-collection-table.html

```diff
@@ -71,8 +71,13 @@
                             <tr class="complex-panel-nested-field" ccdConditionalShow ccdLabelSubstitutor [caseField]="field"
                                 [contextFields]="vLabel.value.caseField  | ccdFieldsFilter">
                               <th><span class="text-16">{{field.label}}</span></th>
-                              <td>
-                                <span class="text-16"><ccd-field-read [caseField]="field" [context]="context"></ccd-field-read></span>
+                              <td *ngIf="!name.value.hasOwnProperty('CaseReference')">
+                                <span class="text-16"><ccd-field-read [caseField]="addValue(field, name.value)" [context]="context"></ccd-field-read></span>
+                              </td>
+                              <td *ngIf="name.value.hasOwnProperty('CaseReference')">
+                                <a href="/v2/case/{{name.value.CaseReference}}" target="_blank" class="caselink">
```

Comment:
Shouldn't we use `case/:jid/:ctid/:cid` rather than `v2/case/:cid`?
The latter will be removed after v2 stories cleanup and the former will remain which is used now when you click through to get to a case from case list.